light-zk.js/src/test-utils/initLookUpTable.ts

```diff
@@ -136,21 +138,21 @@ export async function initLookUpTableTest(
       REGISTERED_VERIFIER_TWO_PDA,
       MINT,
     ];
-
+    extraAccounts = extraAccounts.concat(additonalAccounts);
     if (extraAccounts) {
       for (var i in extraAccounts) {
         addressesToAdd.push(extraAccounts[i]);
       }
     }
 
-    const extendInstruction = AddressLookupTableProgram.extendLookupTable({
-      lookupTable: lookupTableAddress,
-      authority: payerPubkey,
-      payer: payerPubkey,
-      addresses: addressesToAdd,
-    });
+    // const extendInstruction = AddressLookupTableProgram.extendLookupTable({
```

Comment:
I know I'm stubborn / annoying with complaining about this, but can we please stop leaving commented code? If it's useful for debugging - let's leave an upper comment, like:

```
// This is useful for debugging failing tests.
//
// [here goes the code...]
```

Or with any other legit justification, maybe let's go with something like:

```
// The following code is a more sane way of doing XXX, but we cannot
// use it for now, because of YYY.
// TODO(my_nickname): Switch to the following implementation whenever
// possible.
```

Otherwise, I don't think we should leave old code commented. If someone wants to look at the old implementations, that's what git history is made for.

Reply:
actually we can just uncomment, the function works its just duplicate code and this pr is too large to fix it.

Reply:
done